vshard/cfg.lua

```diff
@@ -252,6 +260,14 @@ local function check_sharding(sharding)
                     names[name] = 2
                 end
             end
+            if replica.rebalancer then
```

Comment:
I suppose, we should also forbid setting `rebalancer` option for non `master` instance. Otherwise, we allow the following configuration, which passes all checks, but doesn't start the rebalancer anywhere:
```lua
{
    sharding = {
        ['cbf06940-0790-498b-948d-042b62cf3d29'] = { -- replicaset #1
            replicas = {
                ['8a274925-a26d-47fc-9e1b-af88ce939412'] = {
                    uri = 'storage:storage@127.0.0.1:3301',
                    name = 'storage_1_a',
                    master = true
                },
                ['3de2e3e1-9ebe-4d0d-abb1-26d301b84633'] = {
                    uri = 'storage:storage@127.0.0.1:3302',
                    name = 'storage_1_b',
		    rebalancer = true,
                }
            },
        }, -- replicaset #1
    rebalancer_mode = 'manual',
}
```

Reply:
I intentionally allowed to run it on a replica. Because it doesn't do any writes.

> but doesn't start the rebalancer anywhere

Are you sure? There is a test on it and it passes.

Reply:
My bad. Found it